examples/core/cubemap/app.js

```diff
@@ -1,14 +1,20 @@
 /* global document */
-import {AnimationLoop, GL, TextureCube, Cube, Matrix4, radians} from 'luma.gl';
+import {
+  AnimationLoop, GL, TextureCube, Cube, Matrix4, radians,
+  resetParameters, setParameters
+} from 'luma.gl';
 
 const animationLoop = new AnimationLoop({
   onInitialize: ({gl, canvas}) => {
     addControls(canvas);
 
-    gl.clearColor(0, 0, 0, 1);
-    gl.clearDepth(1);
-    gl.enable(GL.DEPTH_TEST);
-    gl.depthFunc(GL.LEQUAL);
+    resetParameters(gl);
```

Comment:
I am still confused by this. 

Does this mean that calling getContext on a canvas several times returns the **same** context (https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/getContext).

If this is true it would be surprising, and bad, but also  extremely useful, it would give us a sneaky way to get access to the contexts from other libraries.

And if this is true, I think there is even more reason we should call resetParameters in the `AnimationLoop`.